test/WireMock.Net.Tests/FluentAssertions/WireMockAssertionsTests.cs

```diff
@@ -20,9 +23,21 @@ public WireMockAssertionsTests()
         {
             _server = WireMockServer.Start(Port);
             _server.Given(Request.Create().UsingAnyMethod())
-                            .RespondWith(Response.Create().WithStatusCode(200));
+                .RespondWith(Response.Create().WithStatusCode(200));
 
-            _httpClient = new HttpClient { BaseAddress = new Uri($"http://localhost:{Port}") };
+            _httpClient = new HttpClient {BaseAddress = new Uri($"http://localhost:{Port}")};
+        }
+
+        [Fact]
+        public async Task AtAbsoluteUrl_Should_NotThrowWhenAnyCallWasMadeAtAbsoluteUrl()
```

Comment:
I think it's better to rewrite this (and also other tests) in the 'happy case' form (I'm not 100% about the name of the test method....)

So 
``` c#
[Fact]
        public async Task AtAbsoluteUrl_WhenCallWasMadeAbsoluteUrl_ShouldBeOK() // AtAbsoluteUrl_WhenCallWasMadeAbsoluteUrl_IsOK , AtAbsoluteUrl_WhenCallWasMadeAbsoluteUrl_AssertsOK
        {
            await _httpClient.GetAsync("anyurl");

            _server.Should()
                .HaveReceivedACall()
                .AtAbsoluteUrl("http://localhost:42000/anyurl");
        }
```